lib/screens/home_page/home_page.dart

```diff
@@ -15,50 +17,67 @@ class HomePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final todoBloc = ToDoListBloc();
+    final callTodoFetch = todoBloc.fetchTodo();
     final text = S.of(context);
     final size = MediaQuery.of(context).size;
     return Scaffold(
       body: CustomPaint(
         painter: HeaderPainter(),
-        child: Column(
-          children: [
-            Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [Logo()],
-            ),
-            SizedBox(
-              height: size.height * 0.02,
-            ),
-            Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
-                InkWell(
-                    onTap: () {
-                      Navigator.pushReplacementNamed(context, '/login');
-                    },
-                    child: Icon(Icons.logout, color: Colors.white))
-              ],
-            ),
-            SizedBox(
-              height: size.height * 0.25,
-            ),
-            Center(
-              child: Column(
+        child: Container(
+          child: Column(
+            children: [
+              Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [Logo()],
```

Comment:
Do we need to wrap this Widget inside a Row? Why if you just put the widget alone